src/js/common/schemaform/definitions/nonMilitaryJobs.js

```diff
@@ -0,0 +1,24 @@
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+
+import EmploymentPeriodView from '../../../edu-benefits/components/EmploymentPeriodView';
+
+export const schema = commonDefinitions.nonMilitaryJobs;
```

Comment:
Also, yeah, I think you can still name it uiSchema in this file if it's the default export. Whatever file that's importing it can choose whatever name it wants.